src/app/community/malysis/[analysisId]/components/confidence.tsx

```diff
@@ -2,9 +2,14 @@ import { Report_Evidence_Confidence } from "@buf/safedep_api.bufbuild_es/safedep
 
 export default function Confidence({
   confidence,
+  isMalware,
 }: {
   confidence: Report_Evidence_Confidence;
+  isMalware: boolean;
 }) {
+  if (isMalware && confidence !== Report_Evidence_Confidence.HIGH) {
```

Comment:
I don't think we need to change the `confidence`. Even when its possibly malicious we can just show the confidence i.e. `High` / `Medium` / `Low`.

`Possibly Malicious` is just the final decision when `is_malware` is true and `confidence` is not HIGH